CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog 1.0.0].
```

Comment:
I think this is missing the URL

Reply:
It's linked down below, so this link should work (or it does for me!)